packages/js/plugins/http/src/schema.graphql

```diff
@@ -0,0 +1,30 @@
+type Header {
+    key: String!
+    value: String!
+}
+
+type UrlParam {
+    key: String!
+    value: String!
+}
+
+type Response {
+    status: Integer!
+    statusText: String!
+    headers: [Header!]
+    body: String!
```

Comment:
We could have request error or timeout in which case all those stuff would be null and we would just return some error message

Reply:
Also body should probably be nullable?!

Reply:
Another idea, make "get" and "post" ResponseType nullable since wasm module cannot do much with error message and maybe allow users to pass logger in plugin to log out error.